webdriver/tests/bidi/input/perform_actions/pointer_touch.py

```diff
@@ -195,10 +187,14 @@ async def test_touch_pointer_properties_tilt_twist(
         "pointerleave",
     ] == event_types
     assert events[2]["type"] == "pointerdown"
-    assert events[2]["tiltX"] == 21
-    assert events[2]["tiltY"] == -8
+    assert events[2]["altitudeAngle"] == 1.2
```

Comment:
Please note that this test fails in CI: https://community-tc.services.mozilla.com/tasks/ak0YL4uhRLW7O9u0ZzLiSA/runs/0/logs/live/public/logs/live.log#L201078

Reply:
That's okay. Chromium is currently broken w.r.t. angles.

Reply:
Not really. I had a closer look and it's also failing for Firefox. The reason here is that [only the PointerEvents v3 which is currently in draft](https://w3c.github.io/pointerevents/#pointerevent-interface) includes both `altitudeAngle` and `azimuthAngle` properties. The [currently recommended version 1 does not include both properties](https://www.w3.org/TR/pointerevents/#pointerevent-interface).

Because that's a draft not all browsers [including Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1823510) will have it implemented. As such I would suggest we remove those two checks for now and only compare the tilt values.

Reply:
I see. Then let's do that.